packages/analytics/src/integrations/Integration.ts

```diff
@@ -59,7 +59,7 @@ class Integration<T extends IntegrationOptions> {
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   track(this: this, data: EventData<TrackTypesValues>): void {
-    throw new Error('Method not implemented');
+    // Do nothing
```

Comment:
I think we should keep this throw here. If this is causing issues on integrations that do not need to track anything, just override the method there to specifically do nothing and with that, the error will not appear.

Reply:
I made this change because as we talked offline, Vitorino would trigger track or onSetUser events, for each active integration, which only one integration using only onSetUser, and the other using only track method.
With this throw, the integration that won't override this track method (the one that won't use the track method), will trigger this error by triggering the track from the main class.

Reply:
reverted to throw, plus added track on missing Forter class (without any functionality associated)